antenna/throttler.py

```diff
@@ -254,10 +259,32 @@ def match_unsupported_product(throttler, data):
     """Match unsupported products."""
     products = throttler.config("products")
     product_name = safe_get(data, "ProductName")
-    is_not_supported = product_name not in products
+    is_not_supported = products and product_name not in products
+
+    if is_not_supported:
+        logger.info("ProductName rejected: %r", product_name)
+        return True
+    return False
```

Comment:
I cleaned up the logic here a little, but it should do the same thing it did before.